nvidia/gmsl_camera/test_result.rst

```diff
@@ -3,4 +3,54 @@
 Test Result
 ###########
 
-TODO
+Cameras without ISP can use **Argus API** to preview the camera's video streaming. 
+
+Or you can use **GStreamer NVArgusCameraSrc plugin** to preview the video streaming.
+
+1. Video Frame Preview by GStreamer.
```

Comment:
There is only 1 step, so 1. might not be necessary.

Reply:
OK